kubernetes-with-temporal/temporal/temporal-worker-container.yaml

```diff
@@ -97,6 +97,8 @@ spec:
                 secretKeyRef:
                   name: retooltemporalsecrets
                   key: postgres_password
+            - name: WORKFLOW_BACKEND_HOST
+              value: http://workflows-api
```

Comment:
this is the worker internal to the temporal server. can we add this envvar to `retool-workflows-worker-container.yaml` instead?

Let's also add the envvar in the template at `kubernetes/retool-workflows-worker-container.yaml`

Reply:
ah that already has it [here](https://github.com/tryretool/retool-onpremise/blob/02a00d934edb7af6122859d7c3c9df0f542b8bd9/kubernetes/retool-workflows-worker-container.yaml#L75)

Do we need any changes in that case or are we good already and can close this PR?

Reply:
seems fine to me then! good catch.